src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+      System.out.println("Excavator started working!!!");
+    }
+
+    @Override
+    public void stopWork() {
+      System.out.println("Excavator stop working!!!");

```

Comment:
There is a grammatical error in the message. It should be "Excavator stopped working!!!" instead of "Excavator stop working!!!" to maintain consistency with the past tense used in the `doWork` method.